recipes/soprano/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "Soprano" %}
+{% set version = "0.8.13b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/CCP-NC/soprano/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: e1d8ab62c4d410ab1052021d7ae645296fec2bb8d63938014584c6aa7038961b
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
```

Upstream lists a few more:
```suggestion
  run:
    - python >=3.6
    - numpy
    - scipy
    - ase
```